Keyboards/LanguageKeyboards/French/Data/autosuggestions.json

```diff
@@ -0,0 +1 @@
+
```

Comment:
Just nit picking but extra unnecessary whitespace 😅

Reply:
I'm honestly really excited too 😊 This is gonna be so great :) :)

Thanks for checking it all!